### DISCUSSION TITLE
Getting LSPosed_mod with riru working under Magisk 23.0?

### DISCUSSION THREAD

**Discussion by TheHippoMan** [mywalkb/LSPosed_mod#12] [Q&A]
I have Magisk 23.0 running on my device (crDroid-v7.25, which is Android 11). 

I'd like to try LSPosed_mod, but the docs say that it works with Magisk 24+.

In discussions with others, I've been told that it's likely that the riru version of LSPosed_mod could work under Magisk 23.0, but I don't want to try this unless someone can confirm that they already indeed have LSPosed_mod working in this environment.

I use my device for work and for some other important tasks, and I can't afford for my device to be out of commission for an extended period of time while I might need to diagnose and fix potential bricking or bootloops. This is why I'm asking whether anyone has real-life experience of LSPosed_mod actually working with riru under Magisk 23.0. If so, that will give me confidence to give it a try.

Thank you very much in advance.

**Comment by mywalkb** [comment]
riru was the previous version working until Magisk 23.
You can use it with no issue.
When Magisk switch version 24 with zygisk, riru is abandoned and new version is created.
riru version doesn't work on magisk 24 and viceversa, but if you keep magisk 23 riru works, naturally you must install the riru's module, which is abandoned also it.
Also in original README is wrong, I should improve it.

**Comment by mywalkb** [comment]
I updated README now should be better understood

**Reply by TheHippoMan** [reply]
Thank you!!!
